sunpy/instr/__init__.py

```diff
@@ -0,0 +1,6 @@
+import warnings
+
+from sunpy.util.exceptions import SunpyDeprecationWarning
+
+warnings.warn("sunpy.instr is deprecated and will be removed in sunpy 3.1. Please use sunkit-instruments",
```

Comment:
```suggestion
warnings.warn("sunpy.instr is deprecated and will be removed in sunpy 3.1. Please use the affiliated sunkit-instruments package",
```

Just to be explicit?

Reply:
Will fix.